DuggaSys/resulted.js

```diff
@@ -317,12 +300,20 @@ function leaves() {
 
 	if (!(ocol == col && odir == dir) || typechanged) {
 		typechanged = false;
-		resort();
+    // This one is only here due to a bug where sometimes you need to sort multiple times to get the correct one.
```

Comment:
SortableTable sorting logic should not be implemented. Maybe the updated sorting logic will eleminate this bug. the new logic can be found in GruppB2019W21